_posts/current/blog/2021-01-29-linaro-engineering-highlights-january-2021.md

```diff
@@ -0,0 +1,79 @@
+  creating Portable and Reproducible Kernel Builds with TuxMake, plus
+  information on a new tool called the Open Source Project Criticality Score.
+  Furthermore, there are articles on the LLVM Code-Size Optimization: Machine
+  Outliner
+
+  and SVE/SVE2 support in LLVM Debugger (LLDB).
+date: 2021-01-29 12:12:46
+image: /assets/images/content/the_world.jpg
+tags:
+  - Engineering Highlights
+  - TuxMake
+  - Open Source Security Foundation
+  - LLVM
+category: blog
+author: jon.burcham@linaro.org
+---
+# [Saving Power & Improving Performance With Dynamic Interconnect Scaling](https://www.linaro.org/blog/history-of-the-interconnect-framework/)
+
+**By Georgi Djakov, Linaro engineer, Kernel Working Group**
+
+{% include image.html path="/assets/images/content/core-eng.jpg" class="small-inline left" alt="Core Engineering Icon" %}
+
+The Interconnect API is a framework for configuring the on-chip interconnects in the system. It provides an API for drivers to express their bandwidth needs when transferring data and interacting with the different hardware blocks in the system. The framework tunes the system for the best power and performance while taking into account the aggregated traffic between the different endpoints. It was merged in January 2019 and is available in the Linux kernel since v5.1. In this article I’ll share more about the history behind it. [Continued here.](https://www.linaro.org/blog/history-of-the-interconnect-framework/)
+
+## Portable and Reproducible Kernel Builds with TuxMake
+
+**By Dan Rue, Linaro Principal Technical Lead**
+{% include image.html path="/assets/images/content/tuxmake.png" class="small-inline left" alt="TuxMake Logo" 
+
+[TuxMake](https://gitlab.com/Linaro/tuxmake) is an open-source project from Linaro that began in May 2020 and is designed to make building Linux kernels easier. It provides a command-line interface and a Python library, along with a full set of curated portable build environments distributed as container images. With TuxMake, a developer can build any supported combination of target architecture, toolchain, kernel configuration, and make targets.
+
+Building a Linux kernel is not difficult. Follow the documentation, install the dependencies, and run a couple of make commands. However, if a developer wants to build for multiple architectures, with multiple toolchains, things get complicated quickly. Most developers and maintainers have a set of custom scripts that they have written and maintained to perform their required set of builds. TuxMake provides a common layer of abstraction to reduce the need for every developer to write their own build scripts. [Continue reading at Linux World News (LWN)](https://lwn.net/Articles/841624/).
+
+## Open Source Project Criticality Score
+
+**By Joakim Bech and Vicky Janicki**
+{% include image.html path="/assets/images/content/open-source-security-foundation-icon.png" class="small-inline left" alt="Open Source Security Foundation icon.png" 
```

Comment:
You're missing the closing `%}`

So should be: 

` {% include image.html path="/assets/images/content/open-source-security-foundation-icon.png" class="small-inline left" alt="Open Source Security Foundation icon" %}`

Reply:
Hi Kyle. Noted with thanks.